15-lhcb-dev.md

```diff
@@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
 
 > ## What if getpack asks for my password 1000 times? {.callout}
 > `getpack` might ask you for your password several times.
-> To avoid this, you can create a kerberos token with
-> ```
-> kinit
+> To avoid this, you need to properly configure your `ssh` following the
```

Comment:
Yes, for lxplus pubkey is disabled so you're forced to use Kerberos (that's
the recommended setup). For svn, you configure it like this and no Kerberos
tokens are needed: just the pubkey. It's quite cool, even if I don't
understand it.

On Thu, Dec 3, 2015 at 11:03 AM, Tim Head <notifications@github.com> wrote:

> In 15-lhcb-dev.md
> <https://github.com/lhcb/first-analysis-steps/pull/159#discussion_r46532543>
> :
>
> > @@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
> >
> >  > ## What if getpack asks for my password 1000 times? {.callout}
> >  > `getpack` might ask you for your password several times.
> > -> To avoid this, you can create a kerberos token with
> > -> ```
> > -> kinit
> > +> To avoid this, you need to properly configure your `ssh` following the
>
> Reading and parsing the config in my head I am now confused why we need
> the keys at all. To log in to lxplus Pubkey is disabled, and for
> svn.cern.ch GSSAPIAuthentication is used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lhcb/first-analysis-steps/pull/159/files#r46532543>.
>



-- 
Dr. Albert Puig Navarro
Laboratoire de Physique des Hautes Energies
Ecole Polytechnique Fédérale de Lausanne (EPFL)
BSP 614.4 (Cubotron UNIL) CH-1015 Lausanne
EPFL Phone: 021 6939808
CERN Phone: 72518